src/main/java/uk/gov/hmcts/reform/logging/exception/ErrorCode.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.reform.logging.exception;
+
+@SuppressWarnings({"squid:S1118", "HideUtilityClassConstructor"})
```

Comment:
why suppress this?

Reply:
after some discussion about other projects we came to solution to better suppress warnings and have better coverage rather than have explicitly declared private untestable constructor methods

Reply:
https://github.com/jacoco/jacoco/pull/529
Fixed in 0.8.0

Reply:
and first one `squid` is for sonarcube

Reply:
good memo ^ for lib update pr in near future